### DISCUSSION TITLE
Send Message using MIME Format Clarification

### DISCUSSION THREAD

**Discussion by dadams39** [microsoftgraph/msgraph-sdk-go#285] [Q&A] [answered]
The SDK shows that it is able to retrieve MIME format in certain aspects:
[Get message](https://learn.microsoft.com/en-us/graph/api/message-get?view=graph-rest-1.0&tabs=http)

but for [Send message](https://learn.microsoft.com/en-us/graph/api/user-sendmail?view=graph-rest-1.0&tabs=go)
There is only the following paragraph:
> Send the message specified in the request body using either JSON or MIME format.
>
> When using JSON format you can include a [file attachment](https://learn.microsoft.com/en-us/graph/api/resources/fileattachment?view=graph-rest-1.0) in the same sendMail action call.
>
> When using MIME format:
>
>Provide the applicable [Internet message headers](https://tools.ietf.org/html/rfc2076) and the [MIME content](https://tools.ietf.org/html/rfc2045), all encoded in base64 format in the request body.
Add any attachments and S/MIME properties to the MIME content.
is shown to be available for certain events....

Could you elaborate on how send content that is in MIME format because there is no code snippet for this?

**Comment by baywet** [comment, marked as answer]
There's currently no support for sending an email using MIME since this is not a concept that can be described with our current metadata pipelines by Outlook.
There are long terms plans to revamp the metadata pipeline and the internal tooling that services use, but it's still a long way and if you want to send emails using MIME types, the workaround is to use a native client at this point. (or the JSON approach which is supported by the SDK)
